File: Second Year/2nd Semester/Medii de Proiectare si Programare/Java/Laborator_1/src/main/java/domain/Referee.java
package domain;

public class Referee extends Entity<Integer>{
    private String username;
    private String password;
    private String first_name;
    private String last_name;
    private String activity;

    public Referee(Integer integer, String username, String password, String first_name, String last_name, String activity) {
        super(integer);
        this.username = username;
        this.password = password;
        this.first_name = first_name;
        this.last_name = last_name;
        this.activity = activity;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getFirst_name() {
        return first_name;
    }

    public void setFirst_name(String first_name) {
        this.first_name = first_name;
    }

    public String getLast_name() {
        return last_name;
    }

    public void setLast_name(String last_name) {
        this.last_name = last_name;
    }

    public String getActivity() {
        return activity;
    }

    public void setActivity(String activity) {
        this.activity = activity;
    }

    @Override
    public String toString() {
        return "Referee{" +
                "username='" + username + '\'' +
                ", password='" + password + '\'' +
                ", first_name='" + first_name + '\'' +
                ", last_name='" + last_name + '\'' +
                ", activity=" + activity +
                '}';
    }
}

File: Second Year/2nd Semester/Medii de Proiectare si Programare/C#/Laborator_1/domain/Referee.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Laborator_1.domain
{
    public class Referee : Entity<int>
    {
        public string Username { get; set; }
        public string Password { get; set; }
        public string First_Name { get; set; }
        public string Last_Name { get; set; }
        public string Activity { get; set; }
        public Referee(string username, string password, string first_name, string last_name, string activity)
        {
            this.Username = username;
            this.Password = password;
            this.First_Name = first_name;
            this.Last_Name = last_name;
            this.Activity = activity;
        }
        public override string ToString()
        {
            return this.Username + ";" + this.Password + ";" + this.First_Name + ";" + this.Last_Name + ";" + this.Activity.ToString();
        }
    }
}
